bindings/python/setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import find_packages, setup, Extension
```

Comment:
Please add our standard LICENSE file and add the license header to all files.

Reply:
LICENSE has been added at the root of the project. Should I add another under the Python bindings? 

Added standard license header to all files.

Reply:
Yes the LICENSE needs to be picked up and distributed with the distribution, eg:https://github.com/mongodb/mongo-python-driver/blob/master/MANIFEST.in#L2

Reply:
Done.